docs/user/07-20-function-processing-stages.md

```diff
@@ -0,0 +1,48 @@
+# Function processing stages
+
+From the moment you [create a Function](/docs/user/03-10-create-inline-function.md) (Function CR) until the time it is ready, it goes through three processing stages that are defined as these condition types:
+
+1. `ConfigurationReady` (PrinterColumn `CONFIGURED`)
+2. `BuildReady` (PrinterColumn `BUILT`)
+3. `Running` (PrinterColumn `RUNNING`)
+
+For a Function to be considered ready, the status of all three conditions must be `True`:  
+
+```bash
+NAME            CONFIGURED   BUILT     RUNNING   RUNTIME    VERSION   AGE
+test-function   True         True      True      nodejs18   1         96s
+```
+
+When you update an existing Function, conditions change asynchronously depending on the change type.  
+
+The diagrams illustrate all three core status changes in the Function processing circle that the Function Controller handles. They also list all custom resources involved in this process and specify in which cases their update is required.
+
+>**NOTE:** Before you start reading, see the [Function CR](/docs/user/06-10-function.md) document for the custom resource detailed definition, the list of all Function's condition types, and reasons for their success or failure.
+
+## Configured
+
+This initial phase starts when you create a Function CR with configuration specifying the Function's setup. It ends with creating a ConfigMap that is used as a building block for a Function image.
+
+![Function configured](../assets/svls-configured.svg)
+
+## Built
+
+This phase involves creating and processing the Job CR. It ends successfully when the Function image is built and sent to the Docker registry. If the image already exists and only an update is required, the Docker image receives a new tag.
+
+Updating an existing Function requires an image rebuild only if you change the Function's body (**source**) or dependencies (**deps**). An update of Function's other configuration details, such as environment variables, replicas, resources, or labels, does not require image rebuild as it only affects the Deployment.
```

Comment:
```suggestion
Updating an existing Function requires an image rebuild only if you change the Function's body (**source**) or dependencies (**deps**). An update of a Function's other configuration details, such as environment variables, replicas, resources, or labels, does not require image rebuild because it only affects the Deployment.
```